Error 1105: cannot convert datum from decimal to type year.

```
 2022/02/10 11:55:19 ddl.go:96: [fatal] [error] [instance 2] ERROR: Error 1105: cannot convert datum from decimal to type year.
/build/schrddl/ddl/dml_ops.go:84: Error when executing SQL: INSERT INTO `8d9b2d60-7831-4216-b46b-26a007b1a7b1` SET `210b0919-1381-4c4f-a7f8-5b14ccd85955` = '-98529740807866132410458.13669', `e13355ee-8164-499b-a0e2-5c0cbf4ed3d8` = '13:54:28', `3f26df0c-5d91-4f81-8081-65c26c63fe14` = '94686079019417559539526305862.33785778944090601497634', `60cb76c0-92bc-402d-8d61-6220e13159fd` = '-31', `8d2c4f8a-0ae6-4c1f-a6dc-a189364fb4f1` = '1.7952315', `230ab63e-5774-42c0-b47b-da9ff1f9d1dd` = '1.0068215097320639', `1c5512df-34fd-47c3-8d8a-9a4bcdf54b95` = '-295593488419338904051946698124831.72', `03e97bec-8171-4695-b6d7-e2a7a9cc237a` = '1.1555924', `defc3e03-32bd-4be0-bbe4-6cd49c0f3cf7` = '1993'
======== DEBUG BEGIN  ========
Dumping expected contents for table `8d9b2d60-7831-4216-b46b-26a007b1a7b1`:
Comment: ea356684-8283-440a-9a10-f2e796315fb3
Charset: gbk, Collate: gbk_chinese_ci
ShardRowId: 0
AutoID: 0
## Non-Primary Indexes: 
Index #0: Name = `a96c13fa-4171-4929-8db2-eb1e1d65411a`, Columnns = [`60cb76c0-92bc-402d-8d61-6220e13159fd`, `34526c07-db5f-45a9-8eca-e344e500d38d`, ]
Index #1: Name = `b8b565aa-95ca-433d-9175-f04c108a8969`, Columnns = [`84abc205-3772-41b0-8b21-beb46ce7fe70`, ]
Index #2: Name = `a26681ba-eaa6-4495-af6e-66d7faa56d30`, Columnns = [`84abc205-3772-41b0-8b21-beb46ce7fe70`, `230ab63e-5774-42c0-b47b-da9ff1f9d1dd`, ]
Index #3: Name = `a04ef857-52d3-4663-bbce-4faebc38350a`, Columnns = [`230ab63e-5774-42c0-b47b-da9ff1f9d1dd`, ]
Index #4: Name = `5bd86153-6554-409c-9d79-a368953bd87b`, Columnns = [`60cb76c0-92bc-402d-8d61-6220e13159fd`, `230ab63e-5774-42c0-b47b-da9ff1f9d1dd`, `510ded8e-c77c-42a5-b483-7aa7267ffcce`, ]
Index #5: Name = `9aa9fa5d-ea9d-45ea-8172-9fd09734dd5f`, Columnns = [`60cb76c0-92bc-402d-8d61-6220e13159fd`, `510ded8e-c77c-42a5-b483-7aa7267ffcce`, `f727c49b-2ec0-451c-9bc8-d9fe9caf481e`, `230ab63e-5774-42c0-b47b-da9ff1f9d1dd`, ]
Index #6: Name = `c0a8789c-1c86-4028-8a4a-aed06a70db62`, Columnns = [`f727c49b-2ec0-451c-9bc8-d9fe9caf481e`, ]
Index #7: Name = `c9212ba2-e494-4973-94f2-345449711842`, Columnns = [`03e97bec-8171-4695-b6d7-e2a7a9cc237a`, ]
Index #8: Name = `01bfb31a-78af-4c68-a165-a852efb427b5`, Columnns = [`8d2c4f8a-0ae6-4c1f-a6dc-a189364fb4f1`, ]
Index #9: Name = `428760bc-272a-4099-987a-aadf89d3ec7c`, Columnns = [`1c5512df-34fd-47c3-8d8a-9a4bcdf54b95`, `60cb76c0-92bc-402d-8d61-6220e13159fd`, `03e97bec-8171-4695-b6d7-e2a7a9cc237a`, `230ab63e-5774-42c0-b47b-da9ff1f9d1dd`, `8d2c4f8a-0ae6-4c1f-a6dc-a189364fb4f1`, `defc3e03-32bd-4be0-bbe4-6cd49c0f3cf7`, `84abc205-3772-41b0-8b21-beb46ce7fe70`, ]
Index #10: Name = `79ed07cf-fb15-4d0c-8a98-9c51bdb9d308`, Columnns = [`210b0919-1381-4c4f-a7f8-5b14ccd85955`, ]
Index #11: Name = `9d01d362-4507-44d2-9529-507b6c8cbf28`, Columnns = [`8d2c4f8a-0ae6-4c1f-a6dc-a189364fb4f1`, `1c5512df-34fd-47c3-8d8a-9a4bcdf54b95`, `60cb76c0-92bc-402d-8d61-6220e13159fd`, `3f26df0c-5d91-4f81-8081-65c26c63fe14`, `230ab63e-5774-42c0-b47b-da9ff1f9d1dd`, ]
Index #12: Name = `a6844078-58b7-412b-b599-8eaea55bb3c4`, Columnns = [`1c5512df-34fd-47c3-8d8a-9a4bcdf54b95`, `03e97bec-8171-4695-b6d7-e2a7a9cc237a`, `8d2c4f8a-0ae6-4c1f-a6dc-a189364fb4f1`, `60cb76c0-92bc-402d-8d61-6220e13159fd`, `3f26df0c-5d91-4f81-8081-65c26c63fe14`, `210b0919-1381-4c4f-a7f8-5b14ccd85955`, `230ab63e-5774-42c0-b47b-da9ff1f9d1dd`, ]
Index #13: Name = `e6ce9350-89a3-40b4-b072-e43ba80a8ddf`, Columnns = [`3f26df0c-5d91-4f81-8081-65c26c63fe14`, ]
Index #14: Name = `d88a255a-b9d9-4a72-88aa-1c8190045bba`, Columnns = [`e13355ee-8164-499b-a0e2-5c0cbf4ed3d8`, ]
Index #15: Name = `72dc2864-e742-480f-893a-1463707b87ef`, Columnns = [`1c5512df-34fd-47c3-8d8a-9a4bcdf54b95`, `210b0919-1381-4c4f-a7f8-5b14ccd85955`, `8d2c4f8a-0ae6-4c1f-a6dc-a189364fb4f1`, `3f26df0c-5d91-4f81-8081-65c26c63fe14`, `defc3e03-32bd-4be0-bbe4-6cd49c0f3cf7`, ]
## Columns: 
Column #0: Name = `e13355ee-8164-499b-a0e2-5c0cbf4ed3d8`, Definition = TIME NULL DEFAULT '00:00:41', isPrimaryKey = false, used in 1 indexes
Column #1: Name = `3f26df0c-5d91-4f81-8081-65c26c63fe14`, Definition = DECIMAL(52,23) NULL DEFAULT '15754494678400820359623412594.42293539496691578136739', isPrimaryKey = false, used in 6 indexes
Column #2: Name = `03e97bec-8171-4695-b6d7-e2a7a9cc237a`, Definition = FLOAT NULL DEFAULT '1.9686818', isPrimaryKey = false, used in 3 indexes
Column #3: Name = `8d2c4f8a-0ae6-4c1f-a6dc-a189364fb4f1`, Definition = FLOAT NULL DEFAULT '1.6364725', isPrimaryKey = false, used in 5 indexes
Column #4: Name = `defc3e03-32bd-4be0-bbe4-6cd49c0f3cf7`, Definition = YEAR NULL DEFAULT '1921', isPrimaryKey = false, used in 2 indexes
Column #5: Name = `230ab63e-5774-42c0-b47b-da9ff1f9d1dd`, Definition = DOUBLE NULL DEFAULT '1.6181464284948879', isPrimaryKey = false, used in 7 indexes
Column #6: Name = `60cb76c0-92bc-402d-8d61-6220e13159fd`, Definition = TINYINT, isPrimaryKey = true, used in 6 indexes
Column #7: Name = `1c5512df-34fd-47c3-8d8a-9a4bcdf54b95`, Definition = DECIMAL(35,2) NULL DEFAULT '-642461596307643168062465110323282.73', isPrimaryKey = false, used in 4 indexes
Column #8: Name = `210b0919-1381-4c4f-a7f8-5b14ccd85955`, Definition = DECIMAL(28,5) NULL DEFAULT '72216155949679509842474.97211', isPrimaryKey = false, used in 7 indexes
## Values (number of rows = 0): 
======== DEBUG END ========
```

--- a/ddl/run.go
+++ b/ddl/run.go
@@ -137,7 +137,8 @@ func dmlIgnoreError(err error) bool {
 		strings.Contains(errStr, "value is out of range in") ||
 		strings.Contains(errStr, "Data Too Long") ||
 		strings.Contains(errStr, "doesn't have a default value") ||
-		strings.Contains(errStr, "specified twice") {
+		strings.Contains(errStr, "specified twice") ||
+		strings.Contains(errStr, "cannot convert datum from decimal to type year") {
 		return true
 	}
 	return false

